examples/bri-3/src/bri/identity/bpiSubjects/agents/bpiSubjects.agent.ts

```diff
@@ -51,6 +51,20 @@ export class BpiSubjectAgent {
     return bpiSubjectToUpdate;
   }
 
+  public async fetchUpdateCandidatesAndThrowIfNoneExist(
+    ids: string[],
+  ): Promise<BpiSubject[]> {
+    const bpiSubjects = await this.storageAgent.getBpiSubjectsById(ids);
+
+    bpiSubjects.forEach((bp) => {
```

Comment:
isn't it more efficient to just compare the length of both collections?

Reply:
Yeah that's how it is on my local. Will push when I am back at computer. @ognjenkurtic

Reply:
Done